app/helpers/application_helper.rb

```diff
@@ -9,6 +9,6 @@ def license_label(id)
 
   # A work uses its thumbnail for preview; file just uses itself.
   def preview_id(presenter)
-    presenter.respond_to?(:thumbnail_id) ? presenter.thumbnail_id : presenter.id
+    presenter.respond_to?(:thumbnail_id) and presenter.thumbnail_id ? presenter.thumbnail_id : presenter.id
```

Comment:
why `and` instead of `&&`? Avoid the need for parens?  I'd rather use the standard `&&` and parens and have it be clear what's going on (I had to stare at this a while and compare it to previous line to get it), but I approve this PR anyway, feel free to merge.